WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -229,18 +229,22 @@ import WordPressShared
     ///     - topic: The topic to show.
     ///
     func showPostsForTopic(topic: ReaderAbstractTopic) {
-        let controller = ReaderStreamViewController.controllerWithTopic(topic)
-        showDetailViewController(controller, sender: self)
+        showDetailViewController(viewControllerForTopic(topic), sender: self)
     }
 
+    private func viewControllerForTopic(topic: ReaderAbstractTopic) -> UIViewController {
```

Comment:
Would we want to type the return here with `ReaderStreamViewController`? The method does seem to infer some kind of generic type though, so no biggie.